encrypted-media-respec.html

```diff
@@ -1661,30 +1663,28 @@
                         <dd>
                           <p>Run the following steps:</p>
                           <ol>
-                            <li><p>Clear all stored session data associated with this object, including the <a def-id="sessionId"></a> and license destruction record.</p>
-                              <p class="note">A subsequent call to <a def-id="load"></a> with the value of this object's <a def-id="sessionId"></a> would fail because there is no data stored for that session ID.</p>
-                            </li>
                             <li>
                               <p>
-                                Run the <a def-id="update-key-statuses-algorithm"></a> algorithm on the <var>session</var>, providing an empty sequence.
+                                Use <var>cdm</var> to clear all stored session data associated with this object, including the <a def-id="sessionId"></a>
+                                and license destruction record.
                               </p>
+                              <p class="note">A subsequent call to <a def-id="load"></a> with the value of this object's <a def-id="sessionId"></a> would fail because there is no data stored for that session ID.</p>
                             </li>
-                            <li><p>Run the <a def-id="session-close-algorithm"></a> algorithm on this object.</p></li>
+                            <li><p>Run the <a def-id="close-session-algorithm"></a> algorithm on this object.</p></li>
```

Comment:
> What does it mean to close a session other than destroy the keys or removing session data (at least from an observable point of view)?

This is a good question. We could say that "closing the session" is identical to "destroying all licenses, keys and other session data that have not been explicitly stored" and then we could consistently use one phrase or the other. Using only the second phrase is inconvenient for the case where it is the CDM that "closed the session".

In this particular algorithm, though, it is *all* session data that is being destroyed, including that which has been explicitly stored. That is a superset of ""destroying all licenses, keys and other session data that have not been explicitly stored" and so if we agree on the above identity then you are right that this closes the session. But this would not be very clear to the reader.

Perhaps we should say above "Close the session and destroy any data which has been stored, including the license destruction record". The  we will call session-closed and perhaps we don't need Close Session at all ?